earthpy/spatial.py

```diff
@@ -267,8 +267,8 @@ def stack(band_paths, out_path="", nodata=None):
                     # make sure value is same data type
                     nodata = np.array([nodata]).astype(arr.dtype)[0]
 
-                # mask the array
-                arr = np.ma.masked_equal(arr, nodata)
+                    # mask the array
+                    arr = np.ma.masked_equal(arr, nodata)
 
                 return arr, prof
```

Comment:
@nkorinek we can merge this today. will you please make a small change to the variable names? i got confused by prof and realized its profile which is really teh meta data. can you please rename it meta throughout and then i will merge this!